File: Modul-4/Soal-5/PRAK405-2410817310012-FAISALTANJUNG.py
n, kelipatan = map(int, input().split())
total = 0

for i in range(1, n + 1):
    hasil_baris = 0

    for j in range(1, i + 1):
        hasil_baris += j * kelipatan

    print("(", end="")
    for j in range(1, i + 1):
        print(f"{j} * {kelipatan}", end="")
        if j < i:
            print(" + ", end="")
    print(f") = {hasil_baris}")

    total += hasil_baris
print(total)

File: Modul-4/Soal-5/PRAK405-2410817310012-FAISALTANJUNG.c
#include <stdio.h>
int main() {
    int n, kelipatan, total = 0;

    printf("");
    scanf("%d %d", &n, &kelipatan);

    for (int i = 1; i <= n; i++) {
        int jumlah = 0;
        for (int j = 1; j <= i; j++) {
            jumlah += j * kelipatan;
        }

        printf("(");
        for (int j = 1; j <= i; j++) {
            printf("%d * %d", j, kelipatan);
            if (j < i) printf(" + ");
        }
        printf(") = %d\n", jumlah);

        total += jumlah;
    }

    printf("%d\n", total);
    return 0;
}
